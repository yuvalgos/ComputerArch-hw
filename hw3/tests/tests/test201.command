./dflow_calc tests/test201.dat tests/test201.in p78 d104 d185 d107 p107 p8 d69 p79 d161 d122 p166 p97 d17 d98 p131 p188 p20 d163 p44 p49 p93 p127 d144 p18 p90 p173 p107 d136 p71 p64 p159 p153 p25 p22 p71 d170 d50 p19 d109 d110 d10 p81 d115 p146 p91 p4 d171 d12 p180 p39 d68 p61 p85 p150 p112 p131 p132 d133 d73 p27 d196 d119 d101 d15 d55 d25 p48 p18 p88 p90 p57 d123 d2 d62 p177 p25 d152 p16 p64 d74 p4 d113 p89 d53 p188 d20 d25 d90 p92 p167 p167 d9 p39 d191 d72 p26 p102 d99 d45 p161 d148 d123 p151 p69 d42 p169 d101 p26 d6 d55 p77 p169 p37 p36 d19 p48 p102 p85 d161 p153 d174 d167 d49 p20 d166 d164 d136