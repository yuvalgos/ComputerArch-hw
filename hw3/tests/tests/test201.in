3 24 1 21
1 0 19 12
0 6 0 0
1 24 25 20
1 6 4 6
3 20 23 24
8 14 31 21
3 21 16 24
7 29 31 28
7 24 3 14
5 19 18 20
8 19 17 22
8 21 0 3
4 26 3 31
8 24 14 3
0 18 23 12
8 18 25 16
4 7 13 0
0 19 18 19
1 20 14 10
0 3 23 3
1 3 28 1
5 22 22 4
5 1 29 19
8 27 24 9
8 16 4 30
3 7 10 12
7 18 12 4
7 2 5 22
2 2 29 19
8 6 29 10
4 10 8 9
0 25 26 11
5 31 5 16
3 29 15 28
1 9 0 10
5 4 21 23
5 2 30 27
2 20 31 22
2 10 7 13
8 25 17 24
0 23 6 9
8 29 23 28
2 11 6 19
5 17 6 21
0 6 24 16
3 19 29 8
4 27 29 12
3 4 29 27
0 25 1 23
7 3 7 27
8 1 8 11
7 2 0 21
8 17 19 7
8 16 14 27
1 21 17 6
8 12 17 8
3 15 25 25
2 9 31 30
0 15 7 14
2 9 13 6
0 4 31 25
8 31 25 16
3 11 0 24
5 15 2 7
7 9 14 12
4 14 1 26
7 29 7 14
7 27 3 5
5 20 20 25
2 18 4 17
2 21 26 1
6 30 10 26
6 10 1 11
5 24 21 14
0 7 20 17
6 7 5 24
6 20 7 12
1 13 22 13
2 26 3 14
8 10 4 20
3 3 31 29
4 13 8 27
8 23 16 10
6 4 27 19
0 9 12 28
8 16 13 14
5 16 23 4
1 22 2 6
0 9 2 23
5 16 20 3
2 19 8 25
3 1 30 13
0 19 15 15
0 3 11 7
1 13 24 5
4 3 12 4
7 0 8 23
0 24 24 28
8 6 7 25
6 15 31 26
2 16 3 21
6 15 3 5
3 15 7 31
1 27 26 22
1 0 17 0
6 3 4 1
4 3 18 29
8 8 24 8
3 13 23 29
1 18 31 29
3 15 2 24
8 15 7 3
3 0 6 18
1 30 30 20
2 20 10 2
6 8 29 0
4 25 27 13
6 25 0 23
7 28 7 28
8 19 14 22
5 24 17 29
2 8 4 28
5 8 28 12
5 23 15 3
2 29 24 31
1 1 22 31
6 27 16 26
2 3 2 23
0 14 16 2
1 17 4 19
1 24 14 13
6 11 19 8
2 2 28 31
7 24 13 23
4 16 29 28
4 6 27 10
2 15 0 17
7 5 1 5
6 23 12 27
3 11 5 14
3 2 18 27
6 10 9 0
1 12 17 16
1 21 5 15
1 27 15 5
6 5 6 15
3 14 4 26
3 17 0 5
1 1 3 17
6 31 9 22
6 22 20 29
8 10 0 17
0 27 28 29
0 28 14 14
2 2 17 1
5 21 14 24
3 29 26 7
8 29 18 28
0 10 14 24
8 9 12 3
0 4 5 0
2 21 28 2
1 9 23 30
1 14 28 2
0 21 3 1
6 2 21 4
4 25 15 4
2 28 17 0
5 17 13 11
4 18 17 2
3 3 6 1
1 18 20 10
2 21 9 30
8 14 9 22
7 9 1 9
3 6 5 11
5 7 8 23
3 11 4 19
7 6 22 10
0 10 16 19
7 0 11 10
2 31 29 25
0 30 19 25
7 20 4 15
6 4 1 10
8 23 29 11
2 18 7 28
0 9 7 25
5 12 21 22
5 27 9 2
3 19 17 16
4 30 24 8
2 13 9 15
3 16 1 23
8 26 28 11
4 9 2 25
0 21 28 1
5 15 27 4
7 28 4 13
