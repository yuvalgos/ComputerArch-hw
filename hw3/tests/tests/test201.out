Reading the opcodes latency info from tests/test201.dat ... Got latency for 9 opcodes
Reading the program file tests/test201.in ... Found 200 instructions
getProgDepth()==249
getDepDepth(78)==13
getInstDeps(104)=={79,88}
getInstDeps(185)=={149,180}
getInstDeps(107)=={70,67}
getDepDepth(107)==102
getDepDepth(8)==0
getInstDeps(69)=={38,49}
getDepDepth(79)==97
getInstDeps(161)=={146,113}
getInstDeps(122)=={84,119}
getDepDepth(166)==206
getDepDepth(97)==183
getInstDeps(17)=={-1,1}
getInstDeps(98)=={74,-1}
getDepDepth(131)==190
getDepDepth(188)==207
getDepDepth(20)==0
getInstDeps(163)=={139,114}
getDepDepth(44)==42
getDepDepth(49)==44
getDepDepth(93)==58
getDepDepth(127)==149
getInstDeps(144)=={138,137}
getDepDepth(18)==23
getDepDepth(90)==137
getDepDepth(173)==186
getDepDepth(107)==102
getInstDeps(136)=={127,80}
getDepDepth(71)==70
getDepDepth(64)==45
getDepDepth(159)==209
getDepDepth(153)==130
getDepDepth(25)==0
getDepDepth(22)==0
getDepDepth(71)==70
getInstDeps(170)=={169,166}
getInstDeps(50)=={26,47}
getDepDepth(19)==23
getInstDeps(109)=={83,67}
getInstDeps(110)=={62,67}
getInstDeps(10)=={-1,5}
getDepDepth(81)==124
getInstDeps(115)=={80,52}
getDepDepth(146)==177
getDepDepth(91)==46
getDepDepth(4)==5
getInstDeps(171)=={136,149}
getInstDeps(12)=={1,-1}
getDepDepth(180)==141
getDepDepth(39)==7
getInstDeps(68)=={50,-1}
getDepDepth(61)==46
getDepDepth(85)==120
getDepDepth(150)==185
getDepDepth(112)==113
getDepDepth(131)==190
getDepDepth(132)==143
getInstDeps(133)=={119,62}
getInstDeps(73)=={51,63}
getDepDepth(27)==0
getInstDeps(196)=={166,167}
getInstDeps(119)=={76,-1}
getInstDeps(101)=={96,71}
getInstDeps(15)=={-1,-1}
getInstDeps(55)=={53,45}
getInstDeps(25)=={-1,-1}
getDepDepth(48)==78
getDepDepth(18)==23
getDepDepth(88)==48
getDepDepth(90)==137
getDepDepth(57)==46
getInstDeps(123)=={119,56}
getInstDeps(2)=={1,1}
getInstDeps(62)=={49,54}
getDepDepth(177)==183
getDepDepth(25)==0
getInstDeps(152)=={113,148}
getDepDepth(16)==0
getDepDepth(64)==45
getInstDeps(74)=={71,66}
getDepDepth(4)==5
getInstDeps(113)=={99,110}
getDepDepth(89)==183
getInstDeps(53)=={46,26}
getDepDepth(188)==207
getInstDeps(20)=={-1,-1}
getInstDeps(25)=={-1,-1}
getInstDeps(90)=={77,81}
getDepDepth(92)==127
getDepDepth(167)==196
getDepDepth(167)==196
getInstDeps(9)=={-1,6}
getDepDepth(39)==7
getInstDeps(191)=={169,135}
getInstDeps(72)=={39,13}
getDepDepth(26)==0
getDepDepth(102)==144
getInstDeps(99)=={76,49}
getInstDeps(45)=={14,25}
getDepDepth(161)==194
getInstDeps(148)=={113,146}
getInstDeps(123)=={119,56}
getDepDepth(151)==165
getDepDepth(69)==46
getInstDeps(42)=={41,-1}
getDepDepth(169)==190
getInstDeps(101)=={96,71}
getDepDepth(26)==0
getInstDeps(6)=={-1,-1}
getInstDeps(55)=={53,45}
getDepDepth(77)==120
getDepDepth(169)==190
getDepDepth(37)==69
getDepDepth(36)==26
getInstDeps(19)=={6,-1}
getDepDepth(48)==78
getDepDepth(102)==144
getDepDepth(85)==120
getInstDeps(161)=={146,113}
getDepDepth(153)==130
getInstDeps(174)=={163,151}
getInstDeps(167)=={137,161}
getInstDeps(49)=={23,41}
getDepDepth(20)==0
getInstDeps(166)=={165,161}
getInstDeps(164)=={154,155}
getInstDeps(136)=={127,80}
